payjoin-cli/Cargo.toml

```diff
@@ -45,10 +45,15 @@ serde = { version = "1.0.160", features = ["derive"] }
 sled = "0.34"
 tokio = { version = "1.38.1", features = ["full"] }
 tokio-rustls = { version = "0.25", features = ["ring"], default-features = false, optional = true }
+toml = "=0.7.3"
```

Comment:
I was able to get this working on `toml = "0.5.11"` It seems like their crate dependency graph is wrong as there are some of the internal deps that require higher msrvs

Reply:
Thank you so much @benalleng